.circleci/config.yml

```diff
@@ -45,83 +40,22 @@ jobs:
           command: |
             git config --global user.email "ci@infinite.red"
             git config --global user.name "Infinite Red"
-      - run:
-          name: Install PNPM for tests
-          command: |
-            sudo chmod 777 /usr/local/bin
-            curl -f https://get.pnpm.io/v6.16.js | node - add --global pnpm@7
-            pnpm config set store-dir ../.pnpm-store
       - restore_cache:
           name: Restore ignite dependency cache
           key: ignite-deps-cache-{{ .Environment.IGNITE_DEPS_PACKAGER }}-{{ checksum "boilerplate/package.json" }}-{{ arch }}-{{ .Environment.IGNITE_DEPS_KEY_SUFFIX }}
       - run:
-          name: Run tests
-          command: yarn ci:test
+          name: Run static tests
+          command: yarn format:check & yarn lint & yarn typecheck
```

Comment:
Using the single `&` means all 3 of these jobs will run in parallel. They are pretty fast, so I think it's not worth the cost of splitting this into another machine

Reply:
I tried wrapping these in one `yarn static-test` script, but for some reason `yarn` will hang a lot even on success

Reply:
Update this to exit with a 1 exit code if any fail